jbpm-test/src/main/java/org/jbpm/test/AbstractBaseTest.java

```diff
@@ -16,19 +16,56 @@
 
 package org.jbpm.test;
 
+import java.util.Properties;
+
 import org.jbpm.process.instance.impl.util.LoggingPrintStream;
+import org.jbpm.test.persistence.util.PersistenceUtil;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.kie.test.util.db.PoolingDataSourceWrapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public abstract class AbstractBaseTest {
-    
+
+    private static final Logger logger = LoggerFactory.getLogger(AbstractBaseTest.class);
+
+    protected static String datasourceName = "jdbc/jbpm-ds";
```

Comment:
Should it be "protected"?
It seems more like a constant (default datasourceName): if it's, there should be private static final and uppercase.
If not, it can be confused in the scope of the methods that uses a param with the same name.

Reply:
Agree. Changed to 
```suggestion
    protected String datasourceName = "jdbc/jbpm-ds";
```

Reply:
I mean this
```suggestion
    protected static final String DATASOURCE_NAME = "jdbc/jbpm-ds";
```

Reply:
Done